web/src/tippyjs.js

```diff
@@ -143,6 +143,35 @@ export function initialize() {
     // below specify the target directly, elements using those should
     // not have the tippy-zulip-tooltip class.
 
+    delegate("body", {
+        target: ".mark-draft-tooltip",
+        delay: [400, 20],
```

Comment:
There are some constants at the top of the file, like `LONG_HOVER_DELAY`, that you should use in lieu of hard-coded values like this.